app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,54 @@
+class ProjectsController < ApplicationController
+  #before_filter :authenticate_admin!, only: [:edit, :create, :destroy,:update]
+
+  def index
+    @projects = Project.all
+  end
+
+  def show
+    @project = Project.find(params[:id])
+    @resources = @project.resources
```

Comment:
you dont need to send @resources to view, you have this by using @project.resources